docs/components/compact.md

```diff
@@ -274,24 +275,24 @@ usage: thanos compact [<flags>]
 Continuously compacts blocks in an object store bucket.
 
 Flags:
-      --block-meta-fetch-concurrency=32  
+      --block-meta-fetch-concurrency=32
```

Comment:
Hi! Thanks for the PR! 

These whitespaces are actually generated by [mdox](https://github.com/bwplotka/mdox), which pastes CLI output directly into these code blocks. Docs CI will fail if these are removed.

You can run `make docs` to correct these and any other markdown formatting issues. :)

Reply:
Thanks for the hint, I yanked the 1st commit from the PR and ran `make docs` to fix some issues with my changes